### PR TITLE
Updating setup.py's metadata

### PR DESCRIPTION
**Pull request by sergiusens**
Added classifiers and license. Updated description, url and version.

Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

### CODE REVIEW

**Comment by sergiusens** [comment]
@dholbach this is an fyi, next time the version is bumped, don't forget about `setup.py` :-)

File: setup.py [hunk 1]
```diff
@@ -36,4 +36,19 @@
...
+      classifiers=(
+        'Development Status :: 4 - Beta',
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'Intended Audience :: System Administrators',
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+        'Natural Language :: English',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
```
Comment (elopio) [inline setup.py]: On the debian control it says ptyhon >= 3.4. Why are you allowing 3.3 here?

Comment (sergiusens) [inline setup.py]: That's a miss, I was waiting for rmadison to return and forgot to update, it should be just 3.4:

    python3 | 3.4.0-0ubuntu2   | trusty          | amd64, arm64, armhf, i386, powerpc, ppc64el
    python3 | 3.4.3-1          | vivid           | amd64, arm64, armhf, i386, powerpc, ppc64el
    python3 | 3.4.3-4ubuntu1   | wily            | amd64, arm64, armhf, i386, powerpc, ppc64el

**Comment by elopio** [comment]
:+1: thank you.

**Pull request merged**